annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -2,5 +2,5 @@ spring.batch.job.enabled=false
 genomenexus.hgvs=<HOST>/variant_annotation/hgvs/
 genomenexus.xrefs=<HOST>/xrefs/
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.hotspot_parameter=summary
+genomenexus.hotspot_mutassessor_parameter=fields=hotspots,mutation_assessor
```

Comment:
this is enrichments parameter right? So maybe call it something like `genomenexus.enrichments_parameters`. Or if you want to specifically just set fields, maybe `genomenexus.enrichments_fields`. And then always include `fields=` in the parameter